test/application_system_test_case.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+require "webdrivers"
+
+Capybara.register_driver :headless_chrome do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+  options.add_argument("--headless") unless ActiveModel::Type::Boolean.new.cast(ENV["HEADFUL"]).in?(%w[y t 1 yes true])
```

Comment:
```suggestion
  options.add_argument("--headless") unless ActiveModel::Type::Boolean.new.cast(ENV["HEADFUL"])
```

This returns `true` or `false`, so we don't need to check if it is included in the given options anymore.

Reply:
Yep. Although not so sure how would we run tests locally with the HEADFUL option set to true. 
I added the options for running the tests like this`HEADFUL=1 rails test test/system/login_test.rb`
If I only run it with  `HEADFUL rails test test/system/login_test.rb` it returns an error. 
Is there a way for running them? What piece am I missing here haha? 🤔

Reply:
If I apply my suggested change (above) I can run the tests "headful" (viewing Chrome) with `HEADFUL=1 rails test:system`.  `HEADFUL=true rails test:system`, among other "truthful" options, also work. `rails test:system`, `HEADFUL=false rails test:system`, and `HEADFUL=0 rails test:system` all run the tests headless.